src/capability/wnfs.ts

```diff
@@ -0,0 +1,162 @@
+import { Capability } from "../types"
+import { capabilities, CapabilityEscalation, CapabilitySemantics } from "../attenuation"
+import { Chained } from "../chained"
+
+
+export const wnfsCapLevels = {
+  "SUPER_USER": 0,
+  "OVERWRITE": -1,
+  "SOFT_DELETE": -2,
```

Comment:
why negative numbers here?

Reply:
Really doesn't matter that much. But I thought that `SUPER_USER` is the ceiling of permissions, so it's the anchor point, thus 0. I.e. I don't expect anything to ever be on level 1.
At first I then made the other permission levels count up, but then the comparison checks are reverse. I.e. I'd expect `wnfsCapLevel[capBig] > wnfsCapLevel[capSmall]` to mean that `capBig` can do more than `capSmall`.
So, I just made them negative :man_shrugging:  :smile: